src/main/java/com/textmessenger/service/MessageServiceImpl.java

```diff
@@ -21,16 +23,21 @@
   private final UserRepository userRepository;
   private final DialogRepository dialogRepository;
   private final NotificationService notificationService;
+  private SimpMessagingTemplate simpMessagingTemplate;
+  private final String wsPath = "/queue/messages";
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Make this final field static too. ([squid:S1170](https://sonarcloud.io/organizations/final_project_text_messenger/rules#rule_key=squid%3AS1170))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=Text_Messenger_Project&pullRequest=175&issues=AWXEj19jlgbYnohYqVxM&open=AWXEj19jlgbYnohYqVxM)

[ISSUE_KEY]: AWXEj19jlgbYnohYqVxM